shiny/templates/chat/llm-enterprise/aws-bedrock-anthropic/template.env

```diff
@@ -0,0 +1,6 @@
+# Once you provided your API key, rename this file to .env
+# The load_dotenv() in the app.py will then load this env variable
```

Comment:
I think it'd be good to include this message in the `shiny create` output. See the `template.json` in the `shiny/templates/apps` section for examples. Here's one:

```json
{
  "type": "app",
  "id": "dashboard",
  "title": "Basic dashboard",
  "description": "A basic, single page dashboard with value boxes, two plots in cards and a sidebar.",
  "next_steps": [
    "Run the app with `shiny run app.py`."
  ],
  "follow_up": [
    {
      "type": "info",
      "text": "Just getting started with Shiny?"
    },
    {
      "type": "action",
      "text": "Learn more at https://shiny.posit.co/py/docs/user-interfaces.html"
    }
  ]
}
```

Reply:
Good call!